### PR TITLE
Handle any exceptions raised when reading files for code segments

### PR DESCRIPTION
**Pull request by twometresteve**
## Goal

Prevent a crash when reading code segments from a file fails.

## Tests

Manually tested locally by forcing an exception and inspecting the stack traces displatyed in the dashboard.

### CODE REVIEW

File: src/Bugsnag/Polyfills.cs [hunk 1]
```diff
@@ -13,7 +13,15 @@ public static IEnumerable<string> ReadLines(string file)
 #if NET35
       return Enumerable.Empty<string>();
 #else
-      return File.ReadLines(file);
+      try
+      {
+        return File.ReadLines(file);
+      }
+      catch (Exception exception)
```
Comment (tomlongridge) [inline src/Bugsnag/Polyfills.cs:20]:
```suggestion
      catch (System.Exception exception)
```

Comment (twometresteve) [reply src/Bugsnag/Polyfills.cs:20]: I noticed that, but across the repo it's already inconsistent and if you've got a `using` clause it is unnecessary to qualify (Resharper, for example, would warn on that).  I considered changing everything in the repo to match, but thought that lightest touch was best at present.

Comment (tomlongridge) [reply src/Bugsnag/Polyfills.cs:20]: No strong view, although I would say that for our codebase Exceptions have more than one meaning.

Comment (twometresteve) [reply src/Bugsnag/Polyfills.cs:20]: Yup, I can see the logic in that.

**Pull request merged**